ddtrace/contrib/azure.functions/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""
```

Comment:
Can we name this integration (and this package) `azure_functions`. This better aligns with out existing conventions